Holo-Host/hp-admin
# D-01079 - Accept pending payments only

### Updates :
 - Ensures that only pending payments are accepted automatically

## To Test :
 - Spin up two agents and sim2h server
 - Have Agent 1 request an amount from Agent 2
 - Have Agent 2 accept that amount (thereby making a promise with an attached request Id)
 - Turn off sim2h
 - Turn back on sim2h
 - See that the automatic acceptance of the payment occurs and both parties have the completed transaction in their history with the correct balance
